pkgs/development/interpreters/metamath/default.nix

```diff
@@ -11,12 +11,18 @@ stdenv.mkDerivation {
   # no permanent link. See discussion at
```

Comment:
This comment can be removed now, right?

Reply:
Ah, yes!

Reply:
I've removed the comment, thanks for spotting it.